eslint.config.js

```diff
@@ -186,32 +170,43 @@ export default [
 				moduleTeardown: false,
 				url: false,
 				q: false,
-				jQuery: true,
-				sinon: true,
-				amdDefined: true,
-				fireNative: true,
-				Globals: true,
-				hasPHP: true,
-				isLocal: true,
-				supportjQuery: true,
-				originaljQuery: true,
-				$: true,
-				original$: true,
-				baseURL: true,
-				externalHost: true
+				jQuery: false,
+				$: false,
+				sinon: false,
+				amdDefined: false,
+				fireNative: false,
+				Globals: false,
+				hasPHP: false,
+				isLocal: false,
+				supportjQuery: false,
+				originaljQuery: false,
+				original$: false,
+				baseURL: false,
+				externalHost: false
 			}
 		},
 		rules: {
 			...jqueryConfig.rules,
-			strict: [ "error", "function" ],
 
-			// See https://github.com/eslint/eslint/issues/2342
-			"no-unused-vars": "off",
+			"no-unused-vars": [
+				"error",
+				{ args: "after-used", argsIgnorePattern: "^_" }
+			],
 
 			// Too many errors
 			"max-len": "off",
-			camelcase: "off",
-			"one-var": "off"
+			camelcase: "off"
+		}
+	},
+
+	{
+		files: [
+			"test/unit/core.js"
+		],
+		rules: {
+
+			// Core has several cases where unused vars are expected
+			"no-unused-vars": "off"
```

Comment:
Can't we just prefix them with underscores?

Reply:
No, it's more complicated than that. The vars are used in scope using eval, which eslint doesn't recognize.